src/Components/Products.js

```diff
@@ -0,0 +1,75 @@
+import { Component } from 'react';
+import { Link } from 'react-router-dom';
+import mockData from '../data/mockData';
+import './Products.css';
+
+class Products extends Component {
+  constructor(props) {
+    super(props);
+
+    // add id to mockData
+    let iterator = 1;
+    function addIdentifier(target) {
+      target.id = iterator;
+      iterator++;
+    }
```

Comment:
`target` 이 객체인것으로 이해가 되긴 합니다. 

하지만 `target` 보다는 `itemObj` 나, `productObj` 같이 한눈에 보이는 변수명 부탁드려요.

정리하자면, 

```suggestion
    let currentID  = 1;
    function setProductID(productObj) {
      productObj.id = currentID ;
      currentID ++;
    }
```

가 좋을것 같아요. 의견 부탁드립니다. :)